tests/src/e2e-smoke.spec.ts

```diff
@@ -45,7 +58,7 @@ describe('Basic e2e verification of podman desktop start', async () => {
       // Direct Electron console to Node terminal.
       page.on('console', console.log);
 
-      const window: JSHandle<BrowserWindow> = await electronApp.browserWindow(page);
+      const window: JSHandle<BrowserWindow> = await pdRunner.app.browserWindow(page);
```

Comment:
probably the runner can provide that object as well ?

```
runner.browserWindow()
```

(as page object is already handled by the runner)